app/config/index.js

```diff
@@ -10,15 +10,23 @@ const schema = Joi.object({
   statementReceiverApiVersion: Joi.string().required(),
   statementReceiverEndpoint: Joi.string().required(),
   schedulesArePublished: Joi.boolean().required().default(false),
-  showSfi23PaymentPeriod: Joi.boolean().optional().default(false)
+  showSfi23PaymentPeriod: Joi.boolean().optional().default(false),
+  sfi23QuarterlyStatementEnabled: Joi.boolean().default(false),
+  scheduleEnabled: Joi.boolean().default(false),
+  sendCrmMessageEnabled: Joi.boolean().default(false),
```

Comment:
This is a nice touch. Didn't know you could do that with Joi, but it makes sense that you can define defaults at the schema level.

Reply:
I didn't know that until recently too, but I like having everything false as standard so great that it can be specified as such in JOI